setup.py

```diff
@@ -48,7 +48,7 @@
     "docker",
     "mccabe",
     "pylint",
-    "pytest",
+    "pytest=7.1.3",
```

Comment:
Why?

Reply:
Had issues running the jobs and this was suggested by Alex who was having the same issue.

Reply:
pytest 7.2.0 (latest) is broken.
for now, only solution is to downgrade to 7.1.3

Reply:
@trentmc

Reply:
I see. I thought it was solvable by renaming "py.test" file. OK then, no problem.